### DISCUSSION TITLE
M12 interaction

### DISCUSSION THREAD

**Discussion by Hodlatoor** [ubiquity/business-development#10] [General]
M12 VC interaction. 

What I have found so far is that they are looking for a pitch deck and conversation. Do you think we should send a very specific product pitch? Don't think we are too early. We should have a good idea of metrics to introduce though.  Thoughts? @sergfeldman @pavlovcik

**Comment by Hodlatoor** [comment]
* reached out asking for details. waiting to hear back.

**Comment by sergfeldman** [comment]
> * reached out asking for details. waiting to hear back.

I sent emails to the address indicated on the site 2 times.

The 2nd time I received

Delivery Status Notification (Failure)
MAILER-DAEMON@amazonses.com
An error occurred while trying to deliver the mail to the following recipients:
innovate@m12.vc

**Comment by sergfeldman** [comment]
> o you think we should send a very specific product pitch?

Yes, I think that the pitchdeck for GitHub and M12 should be specifically geared toward the needs of GitHub.
I am preparing this pitchdeck now.

**Comment by 0x4007** [comment]
> Delivery Status Notification (Failure)
> [MAILER-DAEMON@amazonses.com](mailto:MAILER-DAEMON@amazonses.com)
> An error occurred while trying to deliver the mail to the following recipients:
> [innovate@m12.vc](mailto:innovate@m12.vc)

I just changed the email configuration today. I didn't realize that you were set up on the old Amazon Simple Email Service. Let's set up yours and @Draeieg's emails again. Inbound emails remain unaffected.

### Outbound email setup

Basically you just need to set up an "App Password” https://myaccount.google.com/apppasswords the rest of the instructions are clear in this tutorial: https://jhart99.com/cloudflare-outbound-email/#outbound-email

**Reply by sergfeldman** [reply]
> ### Outbound email setup
> Basically you just need to set up an "App Password” https://myaccount.google.com/apppasswords the rest of the instructions are clear in this tutorial: https://jhart99.com/cloudflare-outbound-email/#outbound-email

@pavlovcik I've just checked that I can send from and receive emails to my Ubiquity email address

**Comment by 0x4007** [comment]
> What I have found so far is that they are looking for a pitch deck and conversation. Do you think we should send a very specific product pitch? Don't think we are too early. We should have a good idea of metrics to introduce though. Thoughts? @sergfeldman @pavlovcik

This should be a "discussion" as there is no bounty specification.
